checker/tests/nullness/MisuseProperties.java

```diff
@@ -16,7 +16,9 @@ void propertiesToHashtable(Properties p) {
         p.put("line.separator", null);
         Hashtable h = p;
         // Error, because HashTable value has NonNull bound.
-        //:: error: (argument.type.incompatible) :: warning: [unchecked] unchecked call to put(K,V) as a member of the raw type java.util.Hashtable
+        // TODO: false negative.
```

Comment:
I did this for all TODOs in test cases where fixing type argument inference will address the todo.  The remaining `// TODO: false negative` comments are false negatives from raw types.  I'm not sure if they will/should be fixed by type argument inference.  I could add references to #635 for those.